build-logic/project-plugins/src/main/kotlin/com.hedera.hashgraph.application.gradle.kts

```diff
@@ -38,9 +35,11 @@ val copyLib =
 // Copy built jar into `data/apps` and rename
 val copyApp =
     tasks.register<Copy>("copyApp") {
+        inputs.property("projectName", project.name)
+
         from(tasks.jar)
         into(sdkDir(layout.projectDirectory).dir("data/apps"))
-        rename { "${project.name}.jar" }
+        rename { "${inputs.properties["projectName"]}.jar" }
```

Comment:
This runs during Gradle's execution phase. With config cache enabled, it is no longer allowed to access the "configuration time state" during execution directly. Everything a task needs, has to be declared as `input`. Hence the changes to our custom task setups.